### PR TITLE
Improve modal and slider accessibility

### PR DESCRIPTION
**Pull request by danieldafoe**
- Create Sass mixins file for easily setting media queries
- Create Sass variables file for tracking global variables
- Added breakpoints map to Sass variables file
- Replace deprecated e.which with e.key for modal keyup listener
- Remove .show from modals instead of toggling it

### CODE REVIEW

File: client/src/sass/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @error 'Breakpoint name #{$bp} does not exist in the $breakpoints map.';
+  }
+}
+
+// Target screens below the specified breakpoint
+@mixin media-breakpoint-down($bp) {
+  $dimension: map-get($breakpoints, $bp);
+
+  @if ($dimension) {
+    @media only screen and (max-width: map-get($breakpoints, $bp)) {
```
Comment (elina-codes) [inline client/src/sass/_mixins.scss:22]: You might see some funky behaviour due to the max-width here and the min-width above using the same limit value. Adjusting one of them by 1px should fix it!

File: client/src/sass/index.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import 'variables';
+@import 'mixins';
```
Comment (elina-codes) [inline client/src/sass/index.scss:2]: 🎉

File: client/src/sass/_mixins.scss (hunk 1 above)
Comment (danieldafoe) [reply client/src/sass/_mixins.scss:22]: Good catch! I'll fix it.

File: client/src/js/main.js [hunk 3]
```diff
@@ -248,6 +246,19 @@ const Utils = {
     updateTimeEl.innerHTML = formattedTime;
     var timeRangeEl = document.querySelector("#time-range");
     timeRangeEl.value = (typeof(waitTimeArr[1]) === 'string') ? 10 : waitTimeArr[1];
+
+    placeModal.addEventListener('keyup', (e) => {
+      console.log(e);
```
Comment (danieldafoe) [inline client/src/js/main.js:251]: Whoops. I'll also remove this. 😅

**Pull request merged**